docs/docsite/rst/guide_modulehelper.rst

```diff
@@ -211,9 +216,10 @@ One of the attributes in that metadata marks the variable for output, and MH mak
     There are two ways to prevent that from happening:
 
         #.  Set ``mute_vardict_deprecation = True`` and the deprecation will be silenced. If the module still uses the old ``VarDict``,
-            it will not be able to update to community.general 11.0.0 (Spring 2026) upon its release.
-        #.  Set ``use_old_vardict = False`` to make the MH module use the new ``VarDict`` immediatelly.
-            The new ``VarDict`` and its use is documented and this is the recommended way to handle this.
+            it will not be able to update to community.general 11.0.0 (Spring 2025) upon its release.
+        #.  Set ``use_old_vardict = False`` to make the MH module use the new ``VarDict`` immediately.
+            We strongly recommend you use the new ``VarDict``, for that you make sure to consult its documentation at
+            :ref:`ansible_collections.community.general.docsite.guide_vardict`.
```

Comment:
Regarding VarDict, please don't forget about https://github.com/ansible-collections/community.general/pull/9833#issuecomment-2798867889 :)

Reply:
Thanks for reminding me, it had slipped away from my mind already.